assets/src/edit-story/components/checklistCard/components/learnMoreLink.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Internal dependencies
+ */
+import { LEARN_MORE } from '../../../app/prepublish/newConstants';
+import { THEME_CONSTANTS, Link } from '../../../../design-system';
+
+export function LearnMoreLink() {
+  return (
+    <Link
+      href={'/#' /* figure out what this links to */}
```

Comment:
nice add. 
When this is in use, we'll probs want this as a prop to be dynamic (obviously)

Reply:
yea wasn't sure  exactly what this would be. Assumed it would link to some external AMP docs, but maybe not 🤷 . Can update to whatever we want tho once we know what the functionality of this is